qa/tasks/scripts.py

```diff
@@ -37,4 +38,4 @@ def run(self, remote, script_name, args=[], as_root=True):
             cmd = "sudo " + cmd
         if args:
             cmd += ' ' + ' '.join(map(str, args))
-        remote.run(label=script_name, args=cmd)
+        remote.run(label=script_name, args=cmd, stdout=StringIO())
```

Comment:
You better use _remote.sh_ since it already does this, we need to avoid usage of StringIO for such kind of operations.